docs/docusaurus/docs/cloud/connect/connect_airflow.md

```diff
@@ -74,7 +74,7 @@ Apache Airflow is an orchestration tool that allows you to schedule and monitor
     def run_gx_airflow():
 
         context = gx.get_context()
-        checkpoint = context.get_checkpoint(name = CHECKPOINT_NAME)
+        checkpoint = context.checkpoints.get(name = CHECKPOINT_NAME)
```

Comment:
totally unnecessary to change, but are we not autoformatting docs? the spaces around the `=` look so foreign to me!